apps/asap-server/src/utils/squidex.ts

```diff
@@ -0,0 +1,17 @@
+import { URL } from 'url';
+import { parse, format } from 'date-fns';
+import { cms } from '../config';
+
+export const createURL = (assets: string[]): string[] => {
+  return assets.map((asset) =>
+    new URL(`/api/assets/${cms.appName}/${asset}`, cms.baseUrl).toString(),
+  );
+};
+
+export const parseDate = (date: string): Date => {
```

Comment:
Isn't this just ISO date?

Reply:
Squidex doesn't store miliseconds https://support.squidex.io/t/postponed-support-iso-8601-missing-miliseconds/2191

Reply:
milliseconds are optional in ISO, so when *parsing* it should work? Only when serializing we should not pretend we have more information (`.000`) than we have

Reply:
Our problem is more on sending data to squidex, we get a 400.